juju/juju
# Finish rename of the 'admin' model to 'controller'

Fixes lp#1586298

QA steps:

  * `juju upgrade-juju --upload-tools` should run. You should not see this error:

        ERROR --upload-tools can only be used with the admin model


(Review request: http://reviews.vapour.ws/r/5018/)
